### PR TITLE
Add Bancor ICO

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-09-12 at 4 47 20 pm
screen shot 2017-09-12 at 4 47 39 pm
screen shot 2017-09-12 at 4 47 54 pm
screen shot 2017-09-12 at 4 48 03 pm

### CODE REVIEW

**Comment by MostafaBalata** [comment]
@rudolfix I'm not sure, If I put the correct cap data, and is BTCS_ETHER_CAP should converted to Ether?

File: src/config.js [hunk 1]
```diff
@@ -241,12 +241,72 @@ const filecoin = {
...
+  tokenContract: '0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c',
+  information: {
+    aliasName: 'Bancor Network',
+    website: 'https://www.bancor.network/',
+    logo: 'https://www.bancor.network/static/images/favicon.ico',
+  },
+  events: {
+    Contribution: {
+      args: {
+        tokens: '_amount',
```
Comment (rudolfix) [inline src/config.js]: this is very wrong! please check event definition in Solidity

File: src/config.js [hunk 2]
```diff
@@ -241,12 +241,72 @@ const filecoin = {
...
+  addedBy: 'Mostafa Balata',
+};
+
 let config = {
   ICOs: {
     'kin-smart-contract-not-provided': kin,
     '0xd4FD252d7D2C9479a8d616F510eAC6243B5DDdf9': zrx,
     'filescoin-smart-contract-not-provided': filecoin,
     '0x1d0dcc8d8bcafa8e8502beaeef6cbd49d3affcdc': gnosis,
+    '0xBbc79794599b19274850492394004087cBf89710': bnt,
```
Comment (rudolfix) [inline src/config.js]: put Bancor above gnosis

File: src/config.js (hunk 1 above)
Comment (rudolfix) [reply src/config.js]: Also this is clearly visible that number of tokens issued that is displayed is so different from totalSupply that it should raise suspicion

**Pull request closed without merging**